src/core/Breadcrumbs/Breadcrumbs.tsx

```diff
@@ -0,0 +1,158 @@
+/*---------------------------------------------------------------------------------------------
+ * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+ * See LICENSE.md in the project root for license terms and full copyright notice.
+ *--------------------------------------------------------------------------------------------*/
+import React from 'react';
+import cx from 'classnames';
+import { useTheme } from '../utils/hooks/useTheme';
+import { CommonProps } from '../utils/props';
+import SvgChevronRight from '@itwin/itwinui-icons-react/cjs/icons/ChevronRight';
+import '@itwin/itwinui-css/css/breadcrumbs.css';
+import { useResizeObserver } from '../utils/hooks/useResizeObserver';
+import { useMergedRefs } from '../utils/hooks/useMergedRefs';
+
+export type BreadcrumbsProps = {
+  /**
+   * Index of the currently active breadcrumb.
+   * Defaults to last index in the items array.
+   */
+  currentIndex?: number;
+  /**
+   * Breadcrumb items.
+   */
+  items: JSX.Element[];
+  /**
+   * Specify a custom separator element to show between breadcrumb items.
+   * Defaults to the `SvgChevronRight` icon.
+   */
+  separator?: React.ReactNode;
+} & Omit<CommonProps, 'title'>;
+
+/**
+ * A breadcrumb trail is used as a navigational aid to help users keep track
+ * of their place in the application. It is often placed before a page's main content.
+ *
+ * For accessibility, make sure to provide an aria-label describing the type of navigation.
```

Comment:
Stories code does not have aria-label. Maybe we could add some default value? I see 'Breadcrumb' in w3 example.

Reply:
Maybe 🤔

Reply:
Added

Reply:
Could you remove that 'Primary' aria-label from examples? Maybe dont need to mention anything about it.. or rephrase it somehow, that there is the default one, but you can override.

Reply:
I think it should always be overridden by the user. "Breadcrumb" is not descriptive enough.

Reply:
Well, I find mostly info only with "Breadcrumb". SO I think this is valid value for most of the cases. What are others?

Reply:
Breadcrumb is a generic value. Users need to add extra context, e.g. "Primary breadcrumb" or "Table breadcrumb" or something.

Reply:
Do you have any reference for that?
Usually you should have single breadcrumb in the page, no?

Reply:
> Do you have any reference for that?

I can't find any "guidelines" because they all talk about breadcrumbs in isolation, rather than as a part of the site. But I did find a couple examples.

See `nav` on top of github.com 
![image](https://user-images.githubusercontent.com/9084735/127015450-6516c9c0-bd8e-45ca-8a0c-5a6c66e3bf4a.png)

See breadcrumbs on https://www.harvard.edu/about-harvard/harvard-in-the-community/
![image](https://user-images.githubusercontent.com/9084735/127015526-b3592324-7ada-4d81-a942-584becbfd086.png)

> Usually you should have single breadcrumb in the page, no?

Not necessarily. We already have breadcrumbs in our `Header`. So if this component is used somewhere in the page, it will definitely need some other label.